scripts/install.py

```diff
@@ -114,21 +114,21 @@ def q_select_vel_model(vel_mod_dir):
 
     v_mod_ver_options = []
     for root, dirnames, filenames in os.walk(vel_mod_dir):
-        # returns the folder that contains params_vel.py
-        for filename in fnmatch.filter(filenames, defaults.params_vel):
+        # returns the folder that contains vm_params.py
+        for filename in fnmatch.filter(filenames, defaults.vm_params):
```

Comment:
Same issue. so instead of reading from the workflow.config, get it from qcore.constant?